docs/develop/go/schedules.mdx

```diff
@@ -70,6 +70,13 @@ func main() {
 // ...
 ```
 
+:::tip Schedule Auto-Deletion
+
+Once a Schedule has completed creating all its Workflow Executions, the Temporal Service deletes it since it won’t fire again.
+The Service doesn't guarantee when this removal will happen.
```

Comment:
Suggestion for both: The Service --> Temporal Service

Reply:
Adopted